.wordlist.txt

```diff
@@ -213,6 +216,7 @@ MyEnvironment
 MyST
 NBIO
 NBIOs
+NHWC
```

Comment:
NHWC is a data layout format used in machine learning, particularly for tensors. It stands for N (batch size), H (height), W (width), and C (channels).